### PR TITLE
Cleanup getting started:git config page

### PR DESCRIPTION
**Pull request by kprosise**
Added links mentioned in the todo notes on the page, minor changes made
such as word use and phrasing.

QA steps taken: read through documented, ran spellcheck and linkcheck,
built html and examined rendered content. No changes made to examples,
so the steps were not walked through.

This applies to FFTK 988.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 09b0eeaaff7058d4464be893ca5afdc9e956769d are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1429/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]
LGTM, only one minor comment, which I'm not completely sure, if it does not make sense, please ignore

File: source/getting-started/git-config/index.rst [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 ===============
 
-Pushing to the Git repositories in your FoundriesFactory is as simple as
-configuring Git on your personal machine to use the **api token** you generated
-as part of your :ref:`account creation <gs-signup>`. Once configured, ``git`` will know when are
-connecting to ``source.foundries.io`` and will use this token to authenticate
-you with our Git server.
+Pushing to your repositories with :ref:`FoundriesFactory <ref-factory-definition>`
+is as simple as configuring Git on your computer to use the :ref:`api token <ref-api-access>`
+generated automatically during :ref:`account creation <gs-signup>`.
```
[the remark this answers is not in the document]
Comment (kprosise) [reply source/getting-started/git-config/index.rst:8]: You're right, not sure why I phrased it that way. I will edit.

Comment (kprosise) [inline source/getting-started/git-config/index.rst:8]:
```suggestion
generated automatically during :ref:`account creation <gs-signup>`.
```
```suggestion
you generated during :ref:`account creation <gs-signup>`.
```
@angolini I can't recall all the details from when I signed up, though I know the first page does not mention. Tokens are first mentioned here, and then in the getting started fioctl page. We can remove this bit, though I'm also wondering if we want to have the fioctl page before this one?

Comment (angolini) [reply source/getting-started/git-config/index.rst:8]: I'm not sure about removing (I come back to this page a lot). But I think reordering is a great suggestion!

**Comment by doanac** [comment]
Docs for f21ef9c997f6d255054bf156e3c6341891ebed0b are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1441/docs/artifacts/html/index.html

**Pull request merged**